### PR TITLE
Add support for ansible with python3 on debian

### PR DESCRIPTION
**Pull request by thiagoalmeidasa**
Evaluates ansible_python_version in order to install the right Debian
dependencies.

fixes mprahl/ansible-role-lets-encrypt-route-53#17

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -4,14 +4,23 @@
...
 
-- name: install the required dependencies (Debian)
+- name: install the required dependencies (Debian) python3
+  apt:
+    name:
+    - openssl
+    - python3-boto
+    - python3-openssl
+    state: present
+  when: ansible_os_family == 'Debian' and ansible_python_version is version_compare('3', '>=')
```
Comment (mprahl) [inline tasks/main.yml]: The documentation says that `version_compare` was renamed to `version` in 2.5:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

Could you please use `version` and verify that this works?

File: tasks/main.yml [hunk 2]
```diff
@@ -4,14 +4,23 @@
...
+
+- name: install the required dependencies (Debian) python2
   apt:
     name:
     - openssl
     - python-boto
     - python-openssl
     state: present
-  when: ansible_os_family == 'Debian'
+  when: ansible_os_family == 'Debian' and ansible_python_version is version_compare('3', '<')
```
Comment (mprahl) [inline tasks/main.yml]: Same as above.

Comment (thiagoalmeidasa) [reply tasks/main.yml]: Done.

File: tasks/main.yml (hunk 1 above)
Comment (thiagoalmeidasa) [reply tasks/main.yml]: Done

**Comment by mprahl** [comment]
@thiagoalmeidasa thanks for the PR! I released this in v4.4.1.

**Pull request merged**